Packs/NetskopeV2/Integrations/NetskopeV2ForContributionToMarketplace/NetskopeV2ForContributionToMarketplace.yml

```diff
@@ -0,0 +1,137 @@
+- display: API Key
+  name: api_key
+  required: true
+  type: 4
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+contentitemexportablefields:
+  contentitemfields:
+    fromServerVersion: ""
+    itemVersion: ""
+    packID: 6b70ebe5-ab08-448a-827d-66cb14e3ed81
+    propagationLabels:
+    - all
+    toServerVersion: ""
+description: Block URLs, domains and file hashes.
+display: Netskope v2_for contribution to marketplace
+name: Netskope v2_for contribution to marketplace
+script:
+  commands:
+  - arguments:
+    - description: Name of the URL list
+      name: list_name
+      required: true
+    - description: URLs to add to the list
+      isArray: true
+      name: url
+      required: true
+    description: Add URLs to the Netskope URL block list
+    name: netskopev2-add-url
+    outputs:
+    - contextPath: Netskope.URLList.id
+      type: number
+    - contextPath: Netskope.URLList.name
+      type: string
+    - contextPath: Netskope.URLList.data
+    - contextPath: Netskope.URLList.data.urls
+    - contextPath: Netskope.URLList.data.type
+      type: string
+    - contextPath: Netskope.URLList.modify_by
+      type: string
+    - contextPath: Netskope.URLList.modify_time
+      type: date
+    - contextPath: Netskope.URLList.modify_type
+      type: string
+    - contextPath: Netskope.URLList.pending
+      type: number
+  - arguments:
+    - description: Name of the URL list
+      name: list_name
+      required: true
+    - description: URLs to remove from the list
+      isArray: true
+      name: url
+      required: true
+    description: Remove URLs from the Netskope URL block list
+    name: netskopev2-remove-url
+    outputs:
+    - contextPath: Netskope.URLList.id
+      type: number
+    - contextPath: Netskope.URLList.name
+      type: string
+    - contextPath: Netskope.URLList.data
+    - contextPath: Netskope.URLList.data.urls
+    - contextPath: Netskope.URLList.data.type
+      type: string
+    - contextPath: Netskope.URLList.modify_by
+      type: string
+    - contextPath: Netskope.URLList.modify_time
+      type: date
+    - contextPath: Netskope.URLList.modify_type
+      type: string
+    - contextPath: Netskope.URLList.pending
+      type: number
+  - arguments: []
+    description: Get all applied and pending URL lists
+    name: netskopev2-get-lists
+    outputs:
+    - contextPath: Netskope.List.id
+      type: number
+    - contextPath: Netskope.List.name
+      type: string
+    - contextPath: Netskope.List.data
+    - contextPath: Netskope.List.data.urls
+    - contextPath: Netskope.List.data.type
+      type: string
+    - contextPath: Netskope.List.modify_by
+      type: string
+    - contextPath: Netskope.List.modify_time
+      type: date
+    - contextPath: Netskope.List.modify_type
+      type: string
+    - contextPath: Netskope.List.pending
+      type: number
```

Comment:
Please add description to the outputs

Reply:
id: "URL List ID"
name: "URL List name"
data: "URL List contents"
data.urls: "List of URLs in URL List"
data.type: "URL List type ('exact' or 'regex')"
modify_by: "User which last modified URL List"
modify_time: "Time which URL List was last modified"
modify_type: "URL List modification type ('Created', 'Edited' or 'Deleted')"
pending: "URL List pending status ('1' if pending, '0' if not)"